line-things-dev-board/README.md

```diff
@@ -0,0 +1,29 @@
+# LINE Things Starter for LINE Things dev board
+
+## Requirements
+* [Arduino IDE](https://www.arduino.cc/en/Main/Software)
+* LINE Things dev board
+* Micro-USB to USB Cable
+
+## Installation
+Please ensure you have Arduino IDE installed and the board **disconnected**.
+
+1. Open Arduino IDE
+2. Go into **Preferences**
+3. Add `https://www.adafruit.com/package_adafruit_index.json` as an 'Additional Board Manager URL'
+4. Go to **Boards Manager** from the Tools -> Board menu
+5. Search for nRF52 and install **Adafruit nRF52 by Adafruit** (version 0.8.6).
```

Comment:
Maybe we can update version of Adafruit library. 0.8.6 is old.

Reply:
Changed to version 0.9.3